nodejs/week1/avg.js

```
@@ -0,0 +1,18 @@
+const args = process.argv.slice(2);
+//console.log(typeof args);
+const numbers = args.map(arg => parseFloat(arg));
+
+if(numbers.length === 0){
+  console.log("Please enter some numbers!");
+  process.exit();
+}
+if(numbers.some(isNaN)){
+  console.log("Error: All inputs must be valid numbers.")
+  process.exit();
+}
```

Great error handling. In order to let the user know that the program did not succeed, we should return an error code, for example `process.exit(1);`.